public/version-aliases/go.json

```diff
@@ -16,6 +17,7 @@
   "1.7.x": "1.7.6",
   "1.8.3": "1.8.3",
   "1.8.x": "1.8.3",
-  "1.2": "1.2.2",
+  "1.9": "1.9",
+  "1.9.x": "1.9",
   "go1": "go1"
-}
\ No newline at end of file
+}
```

Comment:
This is a generated file, and I'm seeing that the way it's input is created needs to be updated now that the golang storage bucket has more than 1000 objects 🙀.  I'll take care of it 👍